src/api/GraphQl/schemas/generated/lens.tsx

```diff
@@ -104,15 +104,6 @@ export type AccessConditionOutput = {
   token?: Maybe<Erc20OwnershipOutput>;
 };
 
-export type AchRequest = {
-  ethereumAddress: Scalars['EthereumAddress'];
-  freeTextHandle?: InputMaybe<Scalars['Boolean']>;
-  handle?: InputMaybe<Scalars['CreateHandle']>;
-  overrideAlreadyClaimed: Scalars['Boolean'];
-  overrideTradeMark: Scalars['Boolean'];
-  secret: Scalars['String'];
-};
```

Comment:
does it make sense to change generated files?

Reply:
These were changed after doing `npm run i` and `npm run codegen`. They should be in .gitignore tbh